deploy/charts/templates/autoscale.yaml

```diff
@@ -0,0 +1,25 @@
+{{- $fullName := include "ui-helm.fullname" . -}}
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "ui-helm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "ui-helm.name" . }}
+    helm.sh/chart: {{ include "ui-helm.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "ui-helm.fullname" . }}
+  minReplicas: {{ .Values.minReplicas }}
+  maxReplicas: {{ .Values.maxReplicas }}
+  metrics:
+    - type: Resource
+      resource:
+        name: memory
+        target:
+          type: AverageValue
+          averageValue: 320Mi
```

Comment:
When the average memory consumption reaches this point, scale up. _This is base don our current app running in cluster mode which has 2 internal node processes. Cluster will soon be removed._